src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -1091,4 +794,85 @@ input TimestampConfiguration {
   createdAt: String
   updatedAt: String
 }
-```
\ No newline at end of file
+```
+
+### Relational directives
+The relational directives are `@hasOne`, `@hasMany`, `@belongsTo` and `@manyToMany`.
+<BlockSwitcher>
+<Block name="@hasOne">
+
+The `@hasOne` will generate:
+
+- Foreign key fields in parent type that refer the the primary key and sort key fields of the child model.
+- Foreign key fields in parent input object of `create` and `update` mutations.
+
+<Callout>
+The foreign key fields can be customized by adding existing fields in `fields` attribute.
+</Callout>
+
+**Type definition of the `@hasOne` directive**
+
+```graphql
+directive @hasOne(fields: [String!]) on FIELD_DEFINITION
+```
+</Block>
+<Block name="@hasMany">
+
+The `@hasMany` will generate:
+
+- Foreign key fields in child type that refer the the primary key and sort key fields of the parent model.
+- Foreign key fields in child input object of `create` and `update` mutations.
+- A global secondary index(GSI) in the child type Amazon DynamoDB table.
+- The default number of nested objects returned is 100. You can override this behavior by setting the limit argument.
```

Comment:
Since this is under the "`@hasMany` will generate:" section, I think it makes sense to not nest this under that.

Reply:
Moved below the type definition